src/pint/models/parameter.py

```diff
@@ -114,9 +114,11 @@ def __init__(
         prior=priors.Prior(priors.UniformUnboundedRV()),
         set_uncertainty=fortran_float,
         use_alias=None,
+        repeatable=False,
```

Comment:
Having this attribute gives us two benefits: 
1. Much easy to identify if a parameter is repeatable. This is a much more intuitive way to do than checking the types.  It makes the ModelBuilder.repeatable_param() much more generic.  
2. They will allow repeating parameters for other types, not just the Masked parameters. 
This is something I want to use in the parameter update.

Reply:
In terms of checking whether parameters are repeatable, providing a .repeatable property, established by inheritance, is a very convenient way of letting the type determine whether a parameter can be repeated.

For a parameter that isn't a mask or prefix parameter, what can it mean that it is repeatable? What meaning could we assign to having `PB 0.9` and `PB 0.8` in the same par file? Surely in order to be usable - that is, to have a usable name within PINT - any such would need to be of a new type, which could then report whether it is repeatable using the method above, rather than adding to the complexity of constructing every Parameter object.

Reply:
I will change .repeatable to a property.